channel.go

```diff
@@ -124,6 +124,13 @@ type ChannelOptions struct {
 	// default handler that delegates to a subchannel.
 	Handler Handler
 
+	// IgnoreMethods allow users to configure TChannel server such that
+	// requests with specified methods can be ignored by the above passed-in handler
+	// and handled natively by TChannel.
+	// Requests with other methods will be handled by passed-in handler.
+	// This is useful for the gradual migration purpose.
+	IgnoreMethods []string
```

Comment:
`Ignore` may be a little unclear (is the handler ignoring these?) perhaps `SkipHandlerMethods []string` ?